AlphaWallet/Browser/ViewControllers/BrowserViewController.swift

```diff
@@ -149,7 +149,7 @@ final class BrowserViewController: UIViewController {
         webView.load(URLRequest(url: url))
     }
 
-    func notifyFinish(callbackID: Int, value: Swift.Result<DappCallback, DAppError>) {
+    func notifyFinish(callbackID: Int, value: Swift.Result<DappCallback, JsonRpcError>) {
```

Comment:
A few lines below this:

```
return "executeCallback(\(callbackID), \"\(error.message)\", null)"
```

Does it need to be changed? If it's not, the new code `-32603` is not passed along right?